packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.ts

```diff
@@ -114,45 +96,42 @@ export const createPluginActions: CreatePluginActions = (
 
   return {
     actions: {
```

Comment:
As seen here, all actions return a promise that resolves when a confirmation of the message has been received.

Reply:
I love how these actions are uniform now.